package.json

```diff
@@ -59,13 +59,13 @@
     "react-hook-form": "^7.17.5",
     "react-transition-group": "^7.17.5"
   },
-  "main": "./src/index.ts",
-  "module": "./src/index.ts",
-  "types": "./src/index.ts",
+  "main": "dist/react-graphql-ui.es.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
   "exports": {
     ".": {
-      "import": "./src/index.ts",
-      "require": "./src/index.ts"
+      "import": "./dist/index.js",
+      "require": "./dist/react-graphql-ui.es.js"
```

Comment:
Is this use of the `dist` folder okay? Is this `devmode` or `webmode`? I just want to make sure we have it `webmode` so it's ready to deploy.